### PR TITLE
doc: Move ceph-deploy docs to doc/install/ceph-deploy

### PR DESCRIPTION
**Pull request by sebastian-philipp**
Signed-off-by: Sebastian Wagner <sebastian.wagner@suse.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: doc/install/index_manual.rst [hunk 1]
```diff
@@ -69,6 +68,4 @@ sequence.
 .. toctree::
    :maxdepth: 2
 
-   Upgrading Ceph <upgrading-ceph>
```
Comment (sebastian-philipp) [inline doc/install/index_manual.rst]: Which is correct: we don't have a documentation to manually upgrade ceph. The link was misleading.

**Comment by liewegas** [comment]
jenkins render docs

**Comment by liewegas** [comment]
this is showing up nested underneath Puppet for some reason:
image

**Comment by liewegas** [comment]
jenkins render docs

**Comment by sebastian-philipp** [comment]
nah, that went wrong:

image

**Comment by sebastian-philipp** [comment]
`doc/start/index.rst` was a ceph-deploy howto. https://docs.ceph.com/docs/master/install/ is now the overview page

**Comment by djgalloway** [comment]
The "Getting Started" link on the homepage is broken now though: https://docs.ceph.com/docs/master/

**Pull request merged**